gamification/src/main/java/juja/microservices/gamification/jersey/UserControllerImpl.java

```diff
@@ -0,0 +1,43 @@
+package juja.microservices.gamification.jersey;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import ua.com.juja.gamification.model.entity.User;
```

Comment:
@sLisnychyi here import ua.com.... but I've found class User in package juja.microservices.gamification.model.entity
This leads to compilation error. Not only here - there lots of places listed in stacktrace below
```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/jersey/UserController.java:[3,44] error: package ua.com.juja.gamification.model.entity does not exist
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/jersey/UserController.java:[11,4] error: cannot find symbol
[ERROR]  interface UserController
/Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/jersey/UserController.java:[13,9] error: cannot find symbol
[ERROR]  interface UserController
/Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/jersey/UserControllerImpl.java:[5,44] error: package ua.com.juja.gamification.model.entity does not exist
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/jersey/UserControllerImpl.java:[6,39] error: package ua.com.juja.gamification.service does not exist
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/jersey/UserControllerImpl.java:[17,12] error: cannot find symbol
[ERROR]  class UserControllerImpl
/Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/jersey/UserControllerImpl.java:[31,11] error: cannot find symbol
[ERROR]  class UserControllerImpl
/Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/jersey/UserControllerImpl.java:[39,16] error: cannot find symbol
[ERROR]  class UserControllerImpl
/Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/service/Service.java:[3,44] error: package ua.com.juja.gamification.model.entity does not exist
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/service/Service.java:[11,4] error: cannot find symbol
[ERROR]  interface Service
/Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/service/Service.java:[13,9] error: cannot find symbol
[ERROR]  interface Service
/Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/service/ServiceImpl.java:[5,44] error: package ua.com.juja.gamification.model.entity does not exist
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/service/ServiceImpl.java:[6,48] error: package ua.com.juja.gamification.model.repository does not exist
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/service/ServiceImpl.java:[14,12] error: cannot find symbol
[ERROR]  class ServiceImpl
/Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/service/ServiceImpl.java:[22,11] error: cannot find symbol
[ERROR]  class ServiceImpl
/Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/service/ServiceImpl.java:[27,16] error: cannot find symbol
[ERROR]  class ServiceImpl
/Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepositoryCustom.java:[3,44] error: package ua.com.juja.gamification.model.entity does not exist
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepositoryCustom.java:[11,4] error: cannot find symbol
[ERROR]  interface UserRepositoryCustom
/Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepositoryCustom.java:[13,9] error: cannot find symbol
[ERROR]  interface UserRepositoryCustom
/Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepositoryImpl.java:[7,44] error: package ua.com.juja.gamification.model.entity does not exist
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepositoryImpl.java:[24,11] error: cannot find symbol
[ERROR]  class UserRepositoryImpl
/Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepositoryImpl.java:[30,16] error: cannot find symbol
[ERROR]  class UserRepositoryImpl
/Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepository.java:[5,44] error: package ua.com.juja.gamification.model.entity does not exist
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepository.java:[9,56] error: cannot find symbol
[ERROR]  class User
/Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepository.java:[13,4] error: cannot find symbol
[ERROR]  interface UserRepository
/Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepository.java:[15,9] error: cannot find symbol
[ERROR]  interface UserRepository
/Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepositoryImpl.java:[18,8] error: cannot find symbol
[ERROR]  class UserRepositoryImpl
/Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepositoryImpl.java:[18,24] error: cannot find symbol
[ERROR]  class UserRepositoryImpl
/Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepositoryImpl.java:[26,44] error: cannot find symbol
[ERROR]  class UserRepositoryImpl
/Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepositoryImpl.java:[31,37] error: cannot find symbol
[INFO] 30 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.434s
[INFO] Finished at: Tue Apr 05 23:16:50 EEST 2016
[INFO] Final Memory: 16M/309M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.3.2:compile (default-compile) on project microservices-gamification: Compilation failure: Compilation failure:
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/jersey/UserController.java:[3,44] error: package ua.com.juja.gamification.model.entity does not exist
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/jersey/UserController.java:[11,4] error: cannot find symbol
[ERROR] interface UserController
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/jersey/UserController.java:[13,9] error: cannot find symbol
[ERROR] interface UserController
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/jersey/UserControllerImpl.java:[5,44] error: package ua.com.juja.gamification.model.entity does not exist
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/jersey/UserControllerImpl.java:[6,39] error: package ua.com.juja.gamification.service does not exist
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/jersey/UserControllerImpl.java:[17,12] error: cannot find symbol
[ERROR] class UserControllerImpl
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/jersey/UserControllerImpl.java:[31,11] error: cannot find symbol
[ERROR] class UserControllerImpl
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/jersey/UserControllerImpl.java:[39,16] error: cannot find symbol
[ERROR] class UserControllerImpl
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/service/Service.java:[3,44] error: package ua.com.juja.gamification.model.entity does not exist
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/service/Service.java:[11,4] error: cannot find symbol
[ERROR] interface Service
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/service/Service.java:[13,9] error: cannot find symbol
[ERROR] interface Service
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/service/ServiceImpl.java:[5,44] error: package ua.com.juja.gamification.model.entity does not exist
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/service/ServiceImpl.java:[6,48] error: package ua.com.juja.gamification.model.repository does not exist
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/service/ServiceImpl.java:[14,12] error: cannot find symbol
[ERROR] class ServiceImpl
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/service/ServiceImpl.java:[22,11] error: cannot find symbol
[ERROR] class ServiceImpl
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/service/ServiceImpl.java:[27,16] error: cannot find symbol
[ERROR] class ServiceImpl
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepositoryCustom.java:[3,44] error: package ua.com.juja.gamification.model.entity does not exist
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepositoryCustom.java:[11,4] error: cannot find symbol
[ERROR] interface UserRepositoryCustom
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepositoryCustom.java:[13,9] error: cannot find symbol
[ERROR] interface UserRepositoryCustom
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepositoryImpl.java:[7,44] error: package ua.com.juja.gamification.model.entity does not exist
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepositoryImpl.java:[24,11] error: cannot find symbol
[ERROR] class UserRepositoryImpl
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepositoryImpl.java:[30,16] error: cannot find symbol
[ERROR] class UserRepositoryImpl
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepository.java:[5,44] error: package ua.com.juja.gamification.model.entity does not exist
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepository.java:[9,56] error: cannot find symbol
[ERROR] class User
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepository.java:[13,4] error: cannot find symbol
[ERROR] interface UserRepository
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepository.java:[15,9] error: cannot find symbol
[ERROR] interface UserRepository
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepositoryImpl.java:[18,8] error: cannot find symbol
[ERROR] class UserRepositoryImpl
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepositoryImpl.java:[18,24] error: cannot find symbol
[ERROR] class UserRepositoryImpl
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepositoryImpl.java:[26,44] error: cannot find symbol
[ERROR] class UserRepositoryImpl
[ERROR] /Users/viktor/Projects/Personal/Juja/sandbox/microservices/gamification/src/main/java/juja/microservices/gamification/model/repository/UserRepositoryImpl.java:[31,37] error: cannot find symbol
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException

Process finished with exit code 1

```